auth-service/src/main/java/com/auth/authservice/domainclientlayer/Mail/MailServiceCall.java

```diff
@@ -0,0 +1,11 @@
+package com.auth.authservice.domainclientlayer.Mail;
+
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
+
+public interface MailServiceCall {
+
+    @POST("/mail")
+    Call<String> sendMail(@Body Mail mail);
```

Comment:
what does @body do?

Reply:
It's sending the request body, basically setting it.